src/utils/types.ts

```diff
@@ -76,3 +76,21 @@ export type TContentfulLink = {
 	openInNewTab: boolean
 	items?: TContentfulLink[]
 }
+
+type ButtonVariants = 'primary' | 'secondary' | 'secondaryInvert' | 'ghost' |
```

Comment:
@Ivana-Jovic ovo nije dobro, BUTTON_VARIANT mora da bude single source of truth za button varijante, znaci ovo mora da bude `keyof typeof BUTTON_VARIANT`

Reply:
@AleksaStevic Ovo je za button color a ne varijant, sad sam ga reimenovala u ButtonColorVariants da ima vise smisla